### ISSUE TITLE
Me: add a content description to the Gravatar for screen readers

### ISSUE THREAD

**Issue by mattmiklic** [wordpress-mobile/WordPress-Android#6092]
The avatar on the Me screen should have a content description set, so that users of screen readers know what it is. I recommend calling it "Profile Photo" to match the label we use in Calypso.
